Header header
float32 fov
string[] object_names
string[] material_names
geometry_msgs/Pose[] object_poses
geometry_msgs/Pose[] object_hit_locations

